lib/cob_index/macros/custom.rb

```diff
@@ -264,7 +264,7 @@ def extract_url_more_links
             label = url_label(f["z"], f["3"], f["y"])
             unless f["u"].nil?
               if f.indicator2 == "2" || NOT_FULL_TEXT.match(label) || !rec.fields("PRT").empty? || f["u"].include?(ARCHIVE_IT_LINKS)
-                unless f["u"].include?("http://library.temple.edu") && (f["u"].include?("scrc") || f["u"].include?("finding_aids"))
+                unless (f["u"].include?("http://library.temple.edu") || f["u"].include?("https://library.temple.edu")) && (f["u"].include?("scrc") || f["u"].include?("finding_aids"))
```

Comment:
@sensei100 is f["u"] an array here or just a string.  Because if it's just a string then using a regex will make easier to grok:

```suggestion
                unless (f["u"].match?(/http[s]*:\/\/library.temple.edu/) && (f["u"].include?("scrc") || f["u"].include?("finding_aids"))
```